.circleci/config.yml

```diff
@@ -16,8 +16,14 @@ jobs:
           name: Code Formatting Check
           command: 'npm run format:check'
       - run:
-          name: Run Unit Tests
-          command: 'npm test'
+          name: Install codecov
```

Comment:
For the similar reason I think it would be even better to move `'npm install -g codecov'` task _after_  `npm run test:coverage'`, but _before_ `'codecov'` - IMO chances are much higher that dev forget to lint code or fix broken unit-test before pushing changes, than failing installation of codecov. If so why even bother installing `codeconv` as coverage only can be generated, _after_ all tests passes?  What do you think? 

```
-run:
         name: Install codecov
         command: 'npm install -g codecov'
      - run:
          name: Upload Coverage Report to Codecov
          command: 'codecov'
```

Reply:
Fixed in 35026310183d723e292cfc2606eb12c3d890c38d

Reply:
Yeah good point